exporter/elasticsearchexporter/README.md

```diff
@@ -126,6 +126,11 @@ behaviours, which may be configured through the following settings:
   - `mode` (default=none): The fields naming mode. valid modes are:
     - `none`: Use original fields and event structure from the OTLP event.
     - `ecs`: Try to map fields to [Elastic Common Schema (ECS)][ECS]
+    - `otel`: Try to map fields defined in the
+          [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions) (version 1.22.0)
```

Comment:
This isn't really mapping the fields (in the sense of rewriting or converting) but stores them as-is, which is in contrast to the ECS mode.

Reply:
It's not quite as is :-) "As is" is "none" mode I think.

Reply:
My main point is that we're not mapping attributes to semantic conventions, the way the ECS mode maps to ECS, and that there's no "trying" involved in it. Why are you saying that this is not as-is? Because of how we treat data_stream attributes?

Reply:
Because strictly  speaking we have a table of mappings for ES documented. 
For example it is not "Body" it is "body_text" or "body_structured".  And other properties are either formatted or renamed (@timestamp) to made fit better to Elastic datastream mappings.
Anyways, let me know how to phrase it better.

Reply:
Updated comment. Let me know if this works better.